### DISCUSSION TITLE
BUG？频道获取管理员列表为空

### DISCUSSION THREAD

**Discussion by XiaoMiku01** [Mrs4s/go-cqhttp#1208] [Q&A]
版本 v1.0.0-beta8-fix1 
最新源码编译的也会复现

复现方式：对/get_guild_members API请求
连接方式：http/反向ws
返回结果为空:{
    "data": {
        "admins": [],
...,    
"retcode": 0,
    "status": "ok"
}

可能是最近加的子频道管理员导致的？

**Comment by Mrs4s** [comment]
还在适配中
